core/blockly.js

```diff
@@ -402,24 +402,24 @@ Blockly.ALIGN_RIGHT = Blockly.constants.ALIGN.RIGHT;
  */
 
 /**
- * @see Blockly.connectionTypes.INPUT_VALUE
+ * @see 
  */
-Blockly.INPUT_VALUE = Blockly.connectionTypes.INPUT_VALUE;
+Blockly.INPUT_VALUE = 
```

Comment:
Failed search-and-replace?  Here and 3x below.

Reply:
😅yes I think here search-and-replace is failing, okay sir I will look into this